packages/storybook/stories/NonReact/NonReactComposites.stories.mdx

```diff
@@ -6,81 +6,142 @@ import { Meta } from '@storybook/addon-docs';
-    token // string
+    displayName: 'DISPLAY_NAME', // string
+    groupId: 'GROUPID', // string
+    userId: 'USERID', // string
+    token: 'ACCCESS_TOKEN' // string
   },
-  elementToRender, // container element
+  containerElement, // container element
   props // Optional, Object contains all optional props for composite, check composite page for more details
 );
 ```
 
-Check [Chat Composite Basic Example](./?path=/docs/composites-chat-basicexample--basic-example) and [Call Composite Basic Example](./?path=/docs/composites-call-basicexample--basic-example) to see optional props of composites
+_[Call Composite Basic Example](./?path=/docs/composites-call-basicexample--basic-example) for details on what additional properties you can apply to the call composite_
 
-For more details of adapter, check [Adapters for Composites](./?path=/story/composite-adapters--page)
+## Complete examples
 
-## Example html code
+Now we can take a look at how this all comes together
 
-Check [Example](https://github.com/Azure/communication-ui-library/blob/PublicPreview/1.0.0-beta.3/samples/StaticHtmlComposites/index.html) for more detailed example of using calling/chat composites
+### Chat html example
 
 ```html
+<meta charset="utf-8" />
 <!-- Replace this url with your own CDN in production environment -->
 <script src="https://github.com/Azure/communication-ui-library/releases/latest/download/chatComposite.js"></script>
 <!-- height need to be set for composite to fit the layout -->
 <div id="chat-container" style="height: 50vh"></div>
 <script type="module">
   const chatAdapter = await chatComposite.loadChatComposite(
     {
-      displayName: displayName,
-      threadId: threadId,
-      endpoint: endpoint,
-      userId: user,
-      token: token
+      displayName: 'DISPLAY_NAME', // string
+      threadId: 'THREAD_ID', // string
+      endpoint: 'ENDPOINT', // string
+      userId: 'USERID', //
+      token: 'TOKEN' // string
     },
     document.getElementById('chat-container'),
     { key: new Date() }
   );
 </script>
 ```
 
+### Calling html example
+
+```html
+<meta charset="utf-8" />
+<!-- Replace this url with your own CDN in production environment -->
+<script src="https://github.com/Azure/communication-ui-library/releases/latest/download/callComposite.js"></script>
+<!-- height need to be set for composite to fit the layout -->
+<div id="call-container" style="height: 50vh"></div>
+<script type="module">
+  const callAdapter = await callComposite.loadCallComposite(
+    {
+      displayName: 'DISPLAY_NAME', // string
+      groupId: 'GROUPID', // string
+      userId: 'USERID', // string
+      token: 'ACCCESS_TOKEN' // string
+    },
+    document.getElementById('call-container'),
+    { key: new Date() }
+  );
+</script>
+```
+
+If you want to see these examples in Github [Examples](https://github.com/Azure/communication-ui-library/blob/PublicPreview/1.0.0-beta.3/samples/StaticHtmlComposites/index.html)
```

Comment:
Weirdly worded sentence. Perhaps: 

```markdown
These examples are also available on [GitHub](...)
```

Also, why do we link to `PublicPreview/1.0.0-beta.3`. Let's update that to https://github.com/Azure/communication-ui-library/blob/1.3.0/samples/StaticHtmlComposites/index.html

But really, we should probably add a new tag `npm-latest` that always tracks the latest stable release and bump that each time we do a stable release.
Then this documentation can link to that.